tuyaha/devices/light.py

```diff
@@ -1,96 +1,162 @@
 from tuyaha.devices.base import TuyaDevice
 
+"""The minimum brightness value set in the API that does not turn off the light."""
+MIN_BRIGHTNESS = 10.3
+BRIGHTNESS_STD_RANGE = (1, 255)
+
+COLTEMP_SET_RANGE = (1000, 10000)
+COLTEMP_KELV_RANGE = (2700, 6500)
+
 
 class TuyaLight(TuyaDevice):
```

Comment:
It looks like you're trying to fix one more unrelated issue. If yes, please, move it to separate PR.

Reply:
Same consideration already done for Climate

Reply:
https://github.com/PaulAnnekov/tuyaha/pull/39#discussion_r499169364

Reply:
As [#39 (comment)](https://github.com/PaulAnnekov/tuyaha/pull/39#discussion_r499169364)